code/modules/mod/modules/modules_engineering.dm

```diff
@@ -260,3 +260,75 @@
 #undef EXTINGUISHER
 #undef NANOFROST
 #undef METAL_FOAM
+
+/obj/item/mod/module/drone
+	name = "MOD Personal Artificial Intelligence Storage"
```

Comment:
You may want to clarify that this is a drone storage, so that people dont put their PAIs in there, and get them deleted without knowing

Reply:
Will change the name, good call